cortex/cli.py

```
@@ -2,25 +2,40 @@
+    Provides interactive installation, configuration management,
+    and package conflict resolution capabilities.
+    """
+    
     def __init__(self):
         self.spinner_chars = ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏']
         self.spinner_idx = 0
+        self.prefs_manager = PreferencesManager()
+        try:
+            self.prefs_manager.load()
+        except Exception:
+            pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)